brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -388,17 +427,16 @@ protected void applyLifecycle(Map<String, Operation> ops, String opKey, EntitySp
                 try {
                     Path csarPath = csarFileRepository.getCSAR(artifact.getArchiveName(), artifact.getArchiveVersion());
                     script = new ResourceUtils(this).getResourceAsString(csarPath.getParent().toString() + "/expanded/" + ref);
-                } catch (CSARVersionNotFoundException e) {
+                } catch (CSARVersionNotFoundException | NullPointerException  e) {
                     script = new ResourceUtils(this).getResourceAsString(ref);
```

Comment:
BTW because this is a script it's probably okay to use `String` but elsewhere we should treat these as `byte[]`  (eg if getting a PNG picture it would break as it stands)